UPGRADE-4.0.md

```diff
@@ -1,6 +1,40 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+Symfony Framework
+-----------------
+
+The first step to upgrade a Symfony 3.x application to 4.x is to update the
+file and directory structure of your application:
+
+| Symfony 3.x                         | Symfony 4.x
+| ----------------------------------- | --------------------------------
+| `app/config/`                       | `config/`
+| `app/config/*.yml`                  | `config/*.yaml` and `config/packages/*.yaml`
+| `app/config/parameters.yml.dist`    | `config/services.yaml` and `.env.dist`
+| `app/config/parameters.yml`         | `config/services.yaml` and `.env`
+| `app/Resources/<BundleName>/views/` | `templates/bundles/<BundleName>/`
+| `app/Resources/`                    | `src/Resources/`
```

Comment:
shouldn't this be `resources/` or something along those lines ? Not really source code related stuff...

Reply:
Yes ... but in the past. In old Symfony apps, we put assets, templates, translations, etc. in that app/Resources/ dir. But in new Symfony apps, assets, templates and translations have their own dir. In src/Resources/ we put "resources" related to code. For example, you could put XSD files there.